src/Uno.UI.Tests/App/Views/MyBehavior.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Windows.UI.Xaml;
+
+namespace Uno.UI.Tests.App.Views
+{
+	public class MyBehavior
+	{
+
+		public static double GetBulbousness(DependencyObject obj)
+		{
+			return (double)obj.GetValue(BulbousnessProperty);
+		}
+
+		public static void SetBulbousness(DependencyObject obj, double value)
+		{
+			obj.SetValue(BulbousnessProperty, value);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Refactor this method to add validation of parameter 'obj' before using it.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4826539)